CONFIGDOC.md

```diff
@@ -8,15 +8,15 @@ You can use the Jenkins&trade; plugin for MATLAB&reg; in freestyle and multi-con
 -  [Set Up Multi-Configuration Project](#set-up-multi-configuration-project)
 
 ## Use MATLAB for Build
-Once you install the plugin on Jenkins, the **Use MATLAB version** checkbox appears in the **Build Environment** section of the project configuration window.
+Once you install the plugin on your Jenkins agent, **Use MATLAB version** appears in the **Build Environment** section of the project configuration window.

```

Comment:
I don't think you install plugins on Jenkins agents. I believe you only install them on the master. I would go back to "on Jenkins" if you want to keep "on". I suppose you could say "on your Jenkins master" but that may be more confusing.

Reply:
I see your point, Mark. We typically use "install on a system/platform".  I think it is better to say:  "Once you install the plugin, ...". I will make this change.